packages/core/src/fhir-to-cda/cda-templates/components/medications.ts

```diff
@@ -196,3 +218,10 @@ function createEntryFromStatement(
     },
   ];
 }
+
+function buildcMedicationCode(code: CdaCodeCe | undefined): string {
+  if (!code) return NOT_SPECIFIED;
```

Comment:
If we the conditionals below are kept simple as they're now, we could just add `?` before the dot (`.`) and only have one place for default return at the end.

Defs NABD, though.